awx/main/models/ha.py

```diff
@@ -293,8 +293,8 @@ def refresh_capacity_fields(self):
             self.cpu_capacity = 0
             self.mem_capacity = 0  # formula has a non-zero offset, so we make sure it is 0 for hop nodes
         else:
-            self.cpu_capacity = get_cpu_effective_capacity(self.cpu)
-            self.mem_capacity = get_mem_effective_capacity(self.memory)
+            self.cpu_capacity = get_cpu_effective_capacity(self.cpu, is_control_plane=bool(self.node_type in (Instance.Types.EXECUTION,)))
+            self.mem_capacity = get_mem_effective_capacity(self.memory, is_control_plane=bool(self.node_type in (Instance.Types.EXECUTION,)))
```

Comment:
Oh this looks reversed :-P

If the node_type is execution, then it is _not_ the control plane.

Reply:
Got it! I think I can fix that. Thank you!